src/components/listing-card.js

```diff
@@ -89,7 +89,9 @@ class ListingCard extends Component {
       o => offerStatusToListingAvailability(o.status) === 'sold'
     )
     const isWithdrawn = status === 'inactive'
-    const showFeaturedBadge = this.props.featured && !isSold && !isPending
+    const showPendingBadge = isPending && !isWithdrawn
+    const showSoldBadge = isSold || isWithdrawn
```

Comment:
Do we actually want to show the Sold badge on withdrawn items ?
Would it rather make sense to hide those listings ? Though if it complicates the logic not worth it - we can implement that in the back-end when we transition to fetching listing IDs from back-end next sprint.

Reply:
Yes, makes sense. Not worth adding complexity.